tests/Block/Service/AbstractBlockServiceTestCase.php

```diff
@@ -11,14 +11,15 @@
  * file that was distributed with this source code.
  */
 
-namespace Sonata\BlockBundle\Test;
+namespace Sonata\BlockBundle\Tests\Block\Service;
```

Comment:
I think the idea here is to make it reusable for other libraries and that is why it is in src and not tests